### PR TITLE
Fix ICaptureDevice.Open for WinPcapDevice

### PR DESCRIPTION
**Pull request by yyjdelete**
Otherwise when using with NPcap 0.8+(compatible mode), an Expection is thrown which say that it is failed to find the device.

### CODE REVIEW

**Comment by yyjdelete** [comment]
I'm not sure what's changed with libpcap, but seems pcap_findalldevs_ex/pcap_open and pcap_findalldevs/pcap_open_live should be used in pair, and "rpcap://" should not be used with pcap_open_live

File: SharpPcap/WinPcap/WinPcapDevice.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@
 using System;
 using System.Text;
 using System.Runtime.InteropServices;
-using System.Threading;
```
Comment (chmorgan) [inline SharpPcap/WinPcap/WinPcapDevice.cs]: Line ending change here? Whitespace changes should be in separate commits.

Comment (yyjdelete) [inline SharpPcap/WinPcap/WinPcapDevice.cs]: The file use mixed line-ending?

**Comment by chmorgan** [comment]
I see the line endings are mixed in some (or many) files. I'll have to see about fixing that.

**Comment by chmorgan** [comment]
@yyjdelete lets wait to see what the npcap guys say about the issue. It looks like a bug that they may fix. That would mean we wouldn't have to work around it here. Afaict the winpcap api docs say that using rpcap:// should work with pcap_open_live(). If that isn't the case then your change looks good, or if npcap is going to step away from supporting that, but it looks like they consider it a bug.

**Comment by hsluoyz** [comment]
Since the issue has been fixed in https://github.com/the-tcpdump-group/libpcap/issues/523, this PR can be closed.
